release.json

```diff
@@ -1,19 +1,20 @@
 {
     "nightly": {
-        "INTEGRATIONS_CORE_VERSION": "master",
-        "OMNIBUS_SOFTWARE_VERSION": "master",
-        "OMNIBUS_RUBY_VERSION": "datadog-5.5.0",
+        "INTEGRATIONS_CORE_VERSION": "7.26.x",
+        "OMNIBUS_SOFTWARE_VERSION": "7.26.x",
+        "OMNIBUS_RUBY_VERSION": "7.26.x",
         "JMXFETCH_VERSION": "0.42.0",
         "JMXFETCH_HASH": "3102eb76aa973e4ac08f9727130979d95eda734876424ca4d92b1c4c43081a1f",
         "MACOS_BUILD_VERSION": "master",
         "WINDOWS_DDNPM_DRIVER": "attestation-signed",
         "WINDOWS_DDNPM_VERSION": "0.98.2.git.41.c46c971",
-        "WINDOWS_DDNPM_SHASUM": "79e641b2078737f6176a50779cb233af6cb2fc86ad6a783c871f1310ff751deb"
+        "WINDOWS_DDNPM_SHASUM": "79e641b2078737f6176a50779cb233af6cb2fc86ad6a783c871f1310ff751deb",
+        "SECURITY_AGENT_POLICIES_VERSION": "master"
```

Comment:
nit: shouldn't this be pinned to `v0.6` instead? I don't think we want the 7.26.x nightlies to pull all updates to the security policies on `master`

Reply:
Do you think I should also create a branch for MACOS_BUILD_VERSION?

Reply:
Probably, but I'm not sure how this value is handled in the build, any thoughts @bkabrda?

Reply:
This is the repo: https://github.com/DataDog/datadog-agent-macos-build

You can use the tag `7.26.0-rc.1`.